SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialConnectionManager.cpp

```diff
@@ -81,6 +110,11 @@ struct ConfigureConnection
 		Params.network.kcp.security_type = WORKER_NETWORK_SECURITY_TYPE_INSECURE;
 		Params.network.tcp.security_type = WORKER_NETWORK_SECURITY_TYPE_INSECURE;
 
+		// Unreal GDK version
+		UnrealGDKVersionPair.name = "gdk_version";
+		UnrealGDKVersionPair.version = TCHAR_TO_ANSI(*GDKVersionLoader::GetGDKVersion());
+		Params.versions = &UnrealGDKVersionPair;
```

Comment:
Oh, didn't even know the worker protocol had functionality for this, nice!
What happens if there is a mismatch between e.g. server and client?
Can this be read somewhere, in e.g. the inspector?

Reply:
This is allegedly logged in shunt (that's what the ticket states).
I guess this would allow us to more easily see when server and client are out of sync. 
What issues do you see arising (in this code) when the client and server are not using the same version?